src/gui/SetupDialog.cpp

```diff
@@ -1471,6 +1509,40 @@ void SetupDialog::setBackgroundArtwork( const QString & _ba )
 
 
 
+void SetupDialog::setAutoSaveInterval( int value )
+{
+	QString minutes = value > 1 ? tr( "minutes" ) : tr( "minute" );
```

Comment:
Cosmetic, but chronologically, I'd put the minute(s) directly above `m_saveIntervalLbl->setText`, since it's nothing to do with `m_saveIntervalSlider`.